AirCasting/ForgotPassword/InterfaceAdapters/EmailResetPasswordService.swift

```diff
@@ -0,0 +1,36 @@
+// Created by Lunar on 15/07/2021.
+//
+
+import Foundation
+
+final class EmailResetPasswordService: ResetPasswordService {
+    
+    private let apiClient: APIClient
+    private let validator: HTTPResponseValidator
+    
+    init(apiClient: APIClient, validator: HTTPResponseValidator) {
+        self.apiClient = apiClient
+        self.validator = validator
+    }
+    
+    func resetPassword(login: String, completion: @escaping (Result<Void, ResetPasswordServiceError>) -> Void) {
+        let params = ["user": ["login": login]]
+        var request = URLRequest(url: URL(string: "http://aircasting.org/users/password.json")!)
```

Comment:
Let's use `URLProvider`

Reply:
I think it is on another branch, not implemented yet :/

Reply:
Let's put a warning with TODO